BUILD.md

```diff
@@ -0,0 +1,128 @@
+# **BUILD**
+
+## **Table of contents** 
+### 1. [**About**](#about)
+### 2. [**Environment**](#environment)
+### 3. [**Development**](#development)
+>#### 3.1. [**Install and Run**](#install-and-run)
+>#### 3.2. [**Style Guide**](#style-guide)
+>#### 3.3. [**Tests**](#security)
+>##### 3.3.1 [**E2E**](#e2e)
+>##### 3.3.2 [**Unitary Tests**](#unitary-tests)
+>#### 3.4 [**Security**](#security)       
+### 4. [**Production**](#production)
+
+
+## **About**
+The **BUILD.md** is a file to check the environment and build specifications of **horusec-cli** project.
+
+
+## **Environment**
+
+- **Golang**: ^1.17.X
```

Comment:
I think that would be good add the GNU Make dependency since is our build system and git and docker, since they are used by Horusec.

I think also that would be nice add a link to home page of these tools

- [Golang: ^1.17.X](https://go.dev/dl/)
- [GNU Make: ^4.2.X](https://www.gnu.org/software/make/)
- [Git: ^2.25.X](https://git-scm.com/)
- [Docker: ^19.3.X](https://docs.docker.com/get-docker/)

Reply:
Why would Git be a development dependency?

Reply:
We use git to get commit author information.

Reply:
Yes, so it would be a usage dependency, not a development dependency?

Reply:
Without Git the tests will fail too.

Reply:
All right